joomla/joomla-cms
# [J4.0] Improve quickicons styling on mobile

### Summary of Changes

Improves the styling of the quickicons on mobile

I've also applied the SCSS coding standards as written here: https://github.com/joomla/coding-standards/pull/146

### Testing Instructions

- apply patch
- ensure there's a pulsating effect
- background will go green or red, depending on whether or not Joomla/Extensions are up to date

### Expected result

![screeny](https://cloud.githubusercontent.com/assets/2019801/22783707/eb3293b2-eec4-11e6-8eb2-403e26ecb85e.png)
